Resync `css/css-multicol` from WPT Upstream

Ahmad-S792 opened the pull request:
#### 2341d4ce6590c1121bf3133f07a01b471a69d03a
<pre>
Resync `css/css-multicol` from WPT Upstream
<a href="https://bugs.webkit.org/show_bug.cgi?id=304099">https://bugs.webkit.org/show_bug.cgi?id=304099</a>
<a href="https://rdar.apple.com/166432714">rdar://166432714</a>

Reviewed by Brandon Stewart.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/bb493cfb216722d620698e9858496cd4c8090d0e">https://github.com/web-platform-tests/wpt/commit/bb493cfb216722d620698e9858496cd4c8090d0e</a>

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/animation/column-rule-width-interpolation-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/animation/column-rule-width-interpolation.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/animation/column-wrap-reset-interpolation-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/animation/column-wrap-reset-interpolation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/animation/columns-interpolation-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/animation/columns-interpolation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/animation/discrete-no-interpolation-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/animation/discrete-no-interpolation.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/animation/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-006.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-007.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-009-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-009-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-013-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-013.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-014-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-014.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-015-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-015.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-016-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-016.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-017-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-017.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-007.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-018-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-018.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-019-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-019.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-007.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-020-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-020.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-021-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-021.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-022-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-022.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-023-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-023.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-024-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-024.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-025-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-025.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-007.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-026-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-026.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-007.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-027-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-027.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-007.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-028-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-028.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-007.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-029-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-029.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-007.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-wrap-no-constraints-001-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-wrap-no-constraints-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-wrap-no-constraints-002-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-wrap-no-constraints-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/columns-shorthand-reset-wrap-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/columns-shorthand-reset-wrap.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/crashtests/block-in-inline-with-spanner-in-overflowed-parent-nested-multicol.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/crashtests/block-in-inline-with-spanner-in-overflowed-parent.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/crashtests/chrome-bug-1301281.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/crashtests/floated-input-in-inline-next-column.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/crashtests/inline-float-parallel-flow.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/crashtests/move-linebreak-to-different-column.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/crashtests/move-newline-pre-text.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/crashtests/oof-in-additional-column-before-spanner.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/crashtests/remove-transform-descendant-becomes-spanner-remove-spanner-child.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/crashtests/spanner-in-relpos-in-abspos-multicol-in-multicol.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/crashtests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/crashtests/zero-column-height.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/multicol-on-broken-image-alt-text-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/multicol-on-broken-image-alt-text.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/multicol-span-all-020-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/multicol-span-all-020-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/multicol-span-all-020.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/offsetProps-002-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/offsetProps-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/parsing/column-rule-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/parsing/column-rule-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/parsing/column-rule-width-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/parsing/column-rule-width-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/parsing/columns-computed-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/parsing/columns-computed.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/parsing/columns-invalid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/parsing/columns-invalid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/parsing/columns-valid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/parsing/columns-valid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/parsing/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/remove-transform-descendant-becomes-spanner-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/remove-transform-descendant-becomes-spanner.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/subpixel-scroll-crash.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/w3c-import.log:

Canonical link: <a href="https://commits.webkit.org/304439@main">https://commits.webkit.org/304439@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/eb866923bdc2b729dc774da1a422a98ec3a17dcc

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/135550 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/159/builds/7928 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/46842 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/143245 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/87233 "Built successfully") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/6d740e90-c8ef-40a0-95c7-2420f86b12c7) 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/155/builds/8565 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/156/builds/7775 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/103588 "Passed tests") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/60/builds/70972 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/482f0df8-684b-416e-9718-9c5ec61788dd) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/138496 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/162/builds/6157 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/121507 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/84457 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/b5f04ea0-9022-4e32-9e7f-793b8104cbfe) 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/154/builds/5933 "Passed tests") | [✅ 🧪 api-mac-debug](https://ews-build.webkit.org/#/builders/165/builds/3546 "Passed tests") | [✅ 🛠 wpe-cairo-libwebrtc](https://ews-build.webkit.org/#/builders/166/builds/3854 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/115152 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/39692 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/145995 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/153/builds/7613 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/40263 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/111952 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/160/builds/7652 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/161/builds/6393 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/112324 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/28500 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/164/builds/5792 "Passed tests") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/117807 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/61556 "Built successfully") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/158/builds/7665 "Built successfully") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/35923 "Passed tests") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/157/builds/7413 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/163/builds/7631 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/152/builds/7513 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

stwrt reviewed (approved):
There are several failing tests that need to be added to TestExpectations.

Ahmad-S792 committed:
Resync `css/css-multicol` from WPT Upstream
https://bugs.webkit.org/show_bug.cgi?id=304099
rdar://166432714

Reviewed by Brandon Stewart.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/bb493cfb216722d620698e9858496cd4c8090d0e

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/animation/column-rule-width-interpolation-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/animation/column-rule-width-interpolation.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/animation/column-wrap-reset-interpolation-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/animation/column-wrap-reset-interpolation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/animation/columns-interpolation-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/animation/columns-interpolation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/animation/discrete-no-interpolation-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/animation/discrete-no-interpolation.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/animation/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-006.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-007.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-009-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-009-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-013-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-013.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-014-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-014.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-015-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-015.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-016-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-016.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-017-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-017.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-007.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-018-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-018.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-019-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-019.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-007.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-020-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-020.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-021-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-021.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-022-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-022.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-023-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-023.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-024-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-024.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-025-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-025.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-007.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-026-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-026.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-007.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-027-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-027.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-007.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-028-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-028.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-007.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-029-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-029.html: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-height-007.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-wrap-no-constraints-001-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-wrap-no-constraints-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-wrap-no-constraints-002-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/column-wrap-no-constraints-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/columns-shorthand-reset-wrap-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/columns-shorthand-reset-wrap.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/crashtests/block-in-inline-with-spanner-in-overflowed-parent-nested-multicol.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/crashtests/block-in-inline-with-spanner-in-overflowed-parent.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/crashtests/chrome-bug-1301281.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/crashtests/floated-input-in-inline-next-column.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/crashtests/inline-float-parallel-flow.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/crashtests/move-linebreak-to-different-column.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/crashtests/move-newline-pre-text.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/crashtests/oof-in-additional-column-before-spanner.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/crashtests/remove-transform-descendant-becomes-spanner-remove-spanner-child.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/crashtests/spanner-in-relpos-in-abspos-multicol-in-multicol.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/crashtests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/crashtests/zero-column-height.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/multicol-on-broken-image-alt-text-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/multicol-on-broken-image-alt-text.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/multicol-span-all-020-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/multicol-span-all-020-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/multicol-span-all-020.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/offsetProps-002-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/offsetProps-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/parsing/column-rule-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/parsing/column-rule-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/parsing/column-rule-width-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/parsing/column-rule-width-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/parsing/columns-computed-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/parsing/columns-computed.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/parsing/columns-invalid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/parsing/columns-invalid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/parsing/columns-valid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/parsing/columns-valid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/parsing/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/remove-transform-descendant-becomes-spanner-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/remove-transform-descendant-becomes-spanner.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/subpixel-scroll-crash.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-multicol/w3c-import.log:

Canonical link: https://commits.webkit.org/304439@main
